playbooks/deploy-ocp-hybrid-multinode.yml

```diff
@@ -233,6 +236,37 @@
       ansible.builtin.import_role:
         name: redhatci.ocp.generate_manifests
 
+    - name: Get stats for each file
+      ansible.builtin.stat:
+        path: "{{ item }}"
+      loop: "{{ extra_manifests }}"
+      register: file_stats
+      when:
+        - extra_manifests is defined
+        - extra_manifests | length > 0
+
+    - name: Assert extra manifests path exists
+      ansible.builtin.assert:
+        that:
+          - item.stat.exists
+        fail_msg: "The file '{{ item.item }}' does not exist."
+      loop: "{{ file_stats.results }}"
+      loop_control:
+        label: "{{ item.item }}"
+      when: file_stats is defined
+
+    - name: Copy extra manifests
+      ansible.builtin.copy:
+        src: "{{ item }}"
+        dest: "{{ repo_root_path }}/generated/{{ cluster_name }}/openshift/{{ item }}"
```

Comment:
Why do we need to save the manifests?

Reply:
For Openshift installer,
This dir is created by `redhatci.ocp.generate_manifests` which then creates an ISO based on the manifest and config in `redhatci.ocp.generate_agent_iso`